pkg/errors/error.go

```diff
@@ -52,20 +52,17 @@ const (
 
 // classifyError classifies given error by type and internals. Successfully classified (not errNoClass) errors don't get reported to sentry.
 func classifyError(err error) errClass {
+	if core.IsErrorCancellation(err) {
```

Comment:
I would move IsErrorCancellation in `errors` (here) and call it from core as errors.IsErrorCancellation

Reply:
errors shouldn't import core? but it does... maybe check how I did `IsConfigureDiagnostic`